src/PointwiseFunctions/AnalyticData/Xcts/BinaryWithGravitationalWaves.hpp

```diff
@@ -0,0 +1,627 @@
+ * \cdot \hat{n}_a)^2] \delta_{ij} + 2 u^iu^j + 3(\vec{u}\cdot\hat{n}_a)^2 - 5
+ * u^2] n_a^i n_a^j + 12 (\vec{u} \cdot \hat{n}_a) u^{(i}n_a^{j)}\Bigr\}_t, \f}
+ *
+ * \f{equation}{
+ * H^{TT\ a}_{ij} [ \vec{u} ; t^{r}_a] = -\frac{1}{r_a(t^{r}_a)} \Bigr\{ [-2u^2
+ * + 2(\vec{u} \cdot \hat{n}_a)^2] \delta^{ij} + 4u^iu^j + [2 u^2 + 2 (\vec{u}
+ * \cdot \hat{n}_a)^2 ] n_a^i n_a^j - 8(\vec{u}\cdot\hat{n}_a) u^{(i}n_a^{j)}
+ * \Bigr\}_{t^{r}_a},
+ * \label{eq:retarded_term} \f}
+ *
+ * and
+ *
+ * \f{align}{
+ * H^{TT\ a}_{ij} [ \vec{u} ; t^{r}_a \to t] &= \nonumber \\
+ *  &- \int^t_{t^{r}_a} d\tau \frac{(t-\tau)}{r_a(\tau)^3}  \Bigr\{ [-5u^2 +
+ * 9(\vec{u} \cdot \hat{n}_a)^2] \delta^{ij} + 6u^iu^j - 12
+ * (\vec{u}\cdot\hat{n}_a) u^{(i}n_a^{j)} + [9 u^2 - 15(\vec{u} \cdot
+ * \hat{n}_a)^2 ]  n_a^i n_a^j\Bigr\} \label{eq:integral_term} \\
+ *  &- \int^t_{t^{r}_a} d\tau \frac{(t-\tau)^3}{r_a(\tau)^5}  \Bigr\{ [u^2 -
+ * 5(\vec{u} \cdot \hat{n}_a)^2] \delta^{ij} + 2 u^iu^j - 20
+ * (\vec{u}\cdot\hat{n}_a) u^{(i}n_a^{j)} + [-5 u^2 + 35(\vec{u} \cdot
+ * \hat{n}_a)^2 ]  n_a^i n_a^j\Bigr\}. \nonumber \f}
+ *
+ * \warning The retarded and integral terms, equations
+ * \f$\eqref{eq:retarded_term}\f$ and \f$\eqref{eq:integral_term}\f$, are not
+ * implemented yet. Instead these terms are set to zero.}
+ *
+ * With this the whole spatial metric is computed up to \f$4PN\f$ order and the
+ * radiative term agrees well with quadrupole predictions. In \cite Tichy2002ec,
+ * the extrinsic curvature is given up to \f$5PN\f$ order by
+ *
+ * \f{equation}{
+ * K^{ij}_{PN} = - \psi^{-10}_{PN} \Bigr[ (\epsilon^3) \tilde{\pi}_{(3)}^{ij} +
+ * (\epsilon^5) \frac{1}{2} \dot{h}^{TT}_{(4)ij} + (\epsilon^5) (\phi_{(2)}
+ * \tilde{\pi}_{(3)}^{ij})^{TT} \Bigr] + O(\epsilon^6). \f}
+ *
+ * where
+ *
+ * \f{equation}{
+ *  \tilde{\pi}_{(3)}^{i j}=\frac{1}{16 \pi} \sum_a p_{a}^k\{-\delta_{i
+ * j}(\frac{1}{r_a})_{, k}+2[\delta_{i k}(\frac{1}{r_a})_{, j}+\delta_{j
+ * k}(\frac{1}{r_a})_{, i}]-\frac{1}{2} r_{a, i j k}\}. \f}
+ */
+class BinaryWithGravitationalWaves
+    : public elliptic::analytic_data::Background,
+      public elliptic::analytic_data::InitialGuess {
+ public:
+  struct MassLeft {
+    static constexpr Options::String help = "The mass of the left black hole.";
+    using type = double;
+  };
+  struct MassRight {
+    static constexpr Options::String help = "The mass of the right black hole.";
+    using type = double;
+  };
+  struct XCoordsLeft {
+    static constexpr Options::String help =
+        "The coordinates on the x-axis of the left black hole.";
+    using type = double;
+  };
+  struct XCoordsRight {
+    static constexpr Options::String help =
+        "The coordinates on the x-axis of the right black hole.";
+    using type = double;
+  };
+  struct YMomentumLeft {
+    static constexpr Options::String help =
+        "The y-axis-componet of the linear momentum of the left black hole.";
+    using type = double;
+  };
+  struct YMomentumRight {
```

Comment:
Is it possible to specify an orbital angular velocity instead of these momenta?

Reply:
We are going to start testing with circular orbits, so this option will desapear. Later, we can then add orbital angular velocity as an option for other orbits in the past evolution. (later merge request)